.circleci/config.yml

```diff
@@ -246,6 +246,10 @@ jobs:
           name: ipynb-check
           command: ./ci/clean_notebooks.py --check ./docs/tutorials
 
+      - run:
+          name: typeignore-check
+          command: ./ci/check_typeignore.py ./src/ ./tests/
```

Comment:
Why not `${SRC_FILES}`? Do we not want to apply this to examples, setup.py, etc? I could see us wanting to be more lax in example code, just some chance we fail to apply it to places we actually want it, especially as we update SRC_FILES over time.